example/infrastructure/src/index.ts

```diff
@@ -202,7 +230,7 @@ const main = async () => {
     }
   );
 
-  // Node 18, provided "aws-sdk"
+  // Node 20, provided "aws-sdk"
   const v2Function = new aws_lambda_nodejs.NodejsFunction(stack, "V2", {
```

Comment:
This handler is also currently broken. Downgrading it to using the Node 16 runtime is not sufficient, as the v2 `aws-sdk` doesn't play nice with ESM. We can either update the runtime version and import statement in `v2.mjs` to something like `'aws-sdk/lib/aws.js'`, or just remove this example

Reply:
Thanks for reporting. We should avoid importing the entire sdk 'aws-sdk/lib/aws.js', is the demo broken some other way?

Reply:
I went with CJS and `require`, it's not ideal but it's working now

Reply:
I'll take a look! It should work with ESM as well no?

Reply:
You're right, thanks. The issue was only with `import DynamoDB from "aws-sdk/clients/dynamodb.js";`, the Lambda can keep the ESM configuration. I've reverted that change 👍